internal/eval/commands.go

```diff
@@ -26,10 +26,10 @@ var (
 		Arity: -1,
 	}
 	echoCmdMeta = DiceCmdMeta{
-		Name:     "ECHO",
```

Comment:
seems like linting issues got fixed when my editor ran format. seems like we don't enforce go fmt in CI